netci.groovy

```diff
@@ -142,3 +142,38 @@ C:\\Python27\\python %WORKSPACE%\\llvm\\tools\\llilc\\test\\llilc_checkpass.py -
     }
   }
 }
+
+[true, false].each { isPR ->
+  ['debug', 'release'].each { configuration ->
+    def newJobName = Utilities.getFullJobName(project, "ubuntu_${configuration}", isPR)
+    
+    def newJob = job (newJobName) {
+      steps {
+        shell('''if which clang-3.5; then
+	export CC=$(which clang-3.5)
+    export CXX=$(which clang++-3.5)
+elif which clang; then
+	export CC=$(which clang)
+    export CXX=$(which clang++) 
+else
+    echo Could not find clang or clang-3.5
+    exit 1
+fi
+
+(cd coreclr && ./build.sh skipmscorlib ${configuration} && cd ..) && (cd llvm && mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=${configuration} -DWITH_CORECLR=../../coreclr/bin/Product/Linux.x64.${configuration} .. && make -j 5)''')
```

Comment:
Updated to reflect how they do it in coreclr (using configuration = Debug/Release, and lowerConfigurations = debug/release)